### DISCUSSION TITLE
My head is not defined

### DISCUSSION THREAD

**Discussion by JohanneBrams** [vercel/next.js#23714] [Help]
Hello, can someone tell me what I did wrong in my HTML code? I followed the guidelines in nextjs-blog to create a new page, but it wont run. 
![Skærmbillede 2021-04-05 kl  17 50 09](https://user-images.githubusercontent.com/81842361/113660248-a4fecc00-96de-11eb-914c-46f742aa4fc6.png)
![Skærmbillede 2021-04-05 kl  17 44 42](https://user-images.githubusercontent.com/81842361/113660255-a7f9bc80-96de-11eb-8cc0-95ed23376c6a.png)

**Comment by imadatyatalah** [comment]
you should import Head component just like that:
`import Head from "next/head"`
